docker-compose-local.yml

```diff
@@ -118,7 +116,7 @@ services:
     environment:
       ALLOW_CONFIG_MUTATIONS: true
       DARTS_PORTAL_URL: http://localhost:3000
-      DARTS_API_URL: http://localhost:4550
+      DARTS_API_URL: http://darts-api:4550
```

Comment:
Yeah this probably makes more sense considering `darts-api` is part of the stack here. I guess we can override the value on the command line somehow if we wish to run against localhost darts-api (i.e when running darts-api in the IDE)?

Reply:
Due to the way docker works with networks, it's not really possible to get a running container to communicate with a service running on the host machine. Unless you have some ideas here?

Reply:
Ah yes you're right. No worries, we can just run darts-portal with yarn as a process on the host if we need to run against darts-api on the host.

Reply:
Yeah indeed, that's how I do it. Or possibly if you run the darts-portal in docker directly it might be work because the networking is different.